umbraco/Umbraco-CMS
# Use OpenIddict with real db instead of inmemory

### Summary 
Removes the EF Core in-memory database and now rely on the real db. This also means a couple of background jobs and middleware have to check for runtime level.

Furthermore we can now use the middelware that gets the host address on the first request, instead of relying on a hardcoded port number.

### Test:
- Ensure backoffice and swagger works for both `Umbraco.Cms.Web.UI` and `Umbraco.Cms.Web.UI.New`
- Ensure swagger login works
- Ensure you can login and restart the server, while your token is still valid (You can continue working in backoffice, without the warnings around tokens)

### Known issues
I found that, if you have a valid token, but removes your connection string, then the installer will complain. This will be handled in a separate PR.
